examples/SLAM/js/hit-test-location-xr8.ts

```diff
@@ -0,0 +1,63 @@
+import { Component, Object as WLEObject, Type } from '@wonderlandengine/api';
+import { vec3 } from 'gl-matrix';
+import { ARSession, XR8Provider, ARProvider } from '../../../';
```

Comment:
You can `npm install ../../../` as descibed in [this tutorial](https://wonderlandengine.com/tutorials/writing-js-library/#testing--debugging). That allows importing from the package name here and the user can easily copy the example and `npm install @wonderlandengine/ar-tracking` over it (or whatever the package will end up being named)

Reply:
will note that for the next PR. Especialy if this is happening what you mentioned in the previous comment `We probably want to split off the general AR framework into its own repo`